datafusion/src/catalog/schema.rs

```diff
@@ -57,6 +57,13 @@ pub trait SchemaProvider: Sync + Send {
             "schema provider does not support deregistering tables".to_owned(),
         ))
     }
+
+    /// If supported by the implementation, checks the table exist in the schema provider or not.
+    /// If no matched table in the schema provider, return false.
+    /// Otherwise, return true.
+    fn table_exist(&self, _name: &str) -> bool {
```

Comment:
nit: we can keep the same as above, use `#[allow(unused_variables)]` instead of ` _`.

Reply:
done, thanks for your comments.